### ISSUE TITLE
distiller

### ISSUE THREAD

**Issue by yardenmatok203** [calico/basenji#168]
Hey,

can you please elaborate your preprocessing with distiller and cooler?

Thanks!

**Comment by gfudenberg** [comment]
Distiller code for going from reads to binned contact maps can be found here:
https://github.com/open2c/distiller-nf
(it relies on pairtools:  https://www.biorxiv.org/content/10.1101/2023.02.13.528389v1.full and cooler)
We used defaults for filters, mapq30 filter for read quality, and set the binsize used for cooler to 2048bp. This was chosen as a multiple of 2 close to the maximum resolution we felt comfortable analyzing the contact datasets in the paper given their read coverage and experimental protocol details.

Use of cooler is integrated into the distiller pipeline. If you already have mapped reads in a .pairs format, though, you can use cooler cload: https://cooler.readthedocs.io/en/latest/cli.html#cooler-cload.

**Comment by yardenmatok203** [comment]
thank you!
Did you used raw files from: https://data.4dnucleome.org/files-fastq/4DNFITHCIIBX/ ?

**Comment by gfudenberg** [comment]
Yes, but for that experiment I'd just start with the .pairs file, as they were processed with distiller anyways.

**Comment by yardenmatok203** [comment]
This will do the ICA and binning?
 please can you tell me which parameters should I use in order it will be close to your processing?

Thank you,
Yarden

**Comment by yardenmatok203** [comment]
I want to use .hic data, what should I use?

thanks a lot!!!
